apps/annotation/mix.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Annotation.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :annotation,
+      version: "0.1.0",
+      build_path: "../../_build",
+      config_path: "../../config/config.exs",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      elixir: "~> 1.10",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      # {:dep_from_hexpm, "~> 0.3.0"},
+      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"},
+      # {:sibling_app_in_umbrella, in_umbrella: true}
```

Comment:
Do we need to add {:emulation, in_umbrella: true} here? I was running the code from a new vagrant

Reply:
Ah you are right! Since we rely on Emulation in Annotation module, we need that line. Lemme add that real quick